cli/rack/__init__.py

```diff
@@ -423,7 +425,17 @@ def go() -> dict:
             go()
         
         if manifest.getPerformOptimization():
-            logger.warning("Optimization requested but not yet implemented")
+            invoke_optimization(optimization_url)
+
+def invoke_optimization(url: Optional[Url]) -> None:
+    @with_status(f'Optimizing triplestore')
+    def go() -> None:
+        url = url or DEFAULT_OPTIMIZE_URL
```

Comment:
Is this fundamentally different than putting a default value on line 430?

Reply:
I did it because it would mean that the rack ui wouldn't break without some fixup since then I could default the argument to None... I don't know if that was the right call, but it's why I did it.